appveyor.yml

```diff
@@ -1,9 +1,15 @@
 ---
+init:
+  # clean path up to just what we need
+  - set PATH=C:\ruby%ruby_version%\bin;C:\Program Files\Git\cmd;C:\Program Files\7-Zip;C:\Program Files\AppVeyor\BuildAgent;C:\Windows\system32;C:\Windows
```

Comment:
Is a specific path necessary for the build to pass? I seems simpler to me to add to the existing path whatever binaries you need.